lisa/util/__init__.py

```diff
@@ -231,3 +234,10 @@ def is_valid_url(url: str, raise_error: bool = True) -> bool:
 
 def filter_ansi_escape(content: str) -> str:
     return __ansi_escape.sub("", content)
+
+
+def dump_file(file_name: Path, content: Any) -> None:
+    file_name.absolute().relative_to(constants.RUN_LOCAL_PATH)
```

Comment:
It's better to add some comments above to specify it's for path validation. after two or three months, we may forget why this line is here.